src/test/java/ibm/jceplus/junit/openjceplus/TestAESGCM.java

```diff
@@ -8,45 +8,17 @@
 
 package ibm.jceplus.junit.openjceplus;
 
-import junit.framework.Test;
-import junit.framework.TestSuite;
+import ibm.jceplus.junit.base.BaseTestAESGCM;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 
-public class TestAESGCM extends ibm.jceplus.junit.base.BaseTestAESGCM {
+@TestInstance(Lifecycle.PER_CLASS)
+public class TestAESGCM extends BaseTestAESGCM {
 
-    //--------------------------------------------------------------------------
-    //
-    //
-    static {
+    @BeforeAll
+    public void beforeAll() {
         Utils.loadProviderTestSuite();
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public TestAESGCM() {
-        super(Utils.TEST_SUITE_PROVIDER_NAME);
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public TestAESGCM(int keySize) throws Exception {
```

Comment:
Do we have a way of trying different key sizes or is that not something we want?